templates/netlify/remix.init/index.js

```diff
@@ -0,0 +1,112 @@
+const inquirer = require("inquirer");
+const fs = require("fs/promises");
+const { join } = require("path");
+const PackageJson = require("@npmcli/package-json");
+
+const { USE_NETLIFY_EDGE } = process.env;
+
+const filesToCopy = [
+  ["README.md"],
+  ["netlify.toml"],
+  ["server.js"],
+  ["remix.config.js"],
+  ["vscode.json", join(".vscode", "settings.json")],
+];
+
+const filesToModify = ["app/entry.server.tsx", "app/root.tsx"];
+
+async function modifyFilesForEdge(files, rootDirectory) {
+  const filePaths = files.map((file) => join(rootDirectory, file));
+  const contents = await Promise.all(
+    filePaths.map((path) => fs.readFile(path, "utf8"))
+  );
+
+  await Promise.all(
+    contents.map((content, index) => {
+      const newContent = content.replace(
+        /@remix-run\/node/g,
+        "@remix-run/deno"
+      );
+      return fs.writeFile(filePaths[index], newContent);
+    })
+  );
+}
+
+async function copyEdgeTemplateFiles(files, rootDirectory) {
+  for (const [file, target] of files) {
+    await fs.copyFile(
+      join(rootDirectory, "remix.init", file),
+      join(rootDirectory, target || file)
+    );
+  }
+}
+
+async function updatePackageJsonForEdge(directory) {
+  const packageJson = await PackageJson.load(directory);
+  const {
+    dependencies: {
+      "@remix-run/netlify": _netlify,
+      "@remix-run/node": _node,
+      ...dependencies
+    },
+    scripts,
+    ...restOfPackageJson
+  } = packageJson.content;
+
+  packageJson.update({
+    // dev script is the same as the start script for Netlify Edge, "cross-env NODE_ENV=production netlify dev"
+    scripts: { ...scripts, dev: scripts["start"] },
+    ...restOfPackageJson,
+    dependencies: {
+      ...dependencies,
+      "@remix-run/netlify-edge": "*",
```

Comment:
`@netlify/edge-functions` is in `peerDependencies` of `@remix-run/netlify-edge`, so we should include it in the template as well.

```suggestion
      "@netlify/edge-functions": "^2.0.0"
      "@remix-run/netlify-edge": "*",
```